os/arch/arm/src/armv7-a/arm_cpupause.c

```diff
@@ -171,7 +171,7 @@ int up_cpu_paused(int cpu)
    * requesting CPU.
    */
 
-  spin_unlock(&g_cpu_paused[cpu]);
+	spin_unlock(&g_cpu_paused[cpu]);
```

Comment:
This file indentation has not been fixed. Pls check again.

Reply:
I will fix the indentation.